expan/core/early_stopping.py

```diff
@@ -260,8 +260,10 @@ def bayes_factor(x, y, distribution='normal', num_iters=25000):
         dictionary with statistics
     """
     traces, n_x, n_y, mu_x, mu_y = _bayes_sampling(x, y, distribution=distribution, num_iters=num_iters)
-    kde = gaussian_kde(traces['delta'])
+    trace_to_evaluate_bf = traces['alpha'] if distribution == 'normal' else traces['delta']
+    trace_to_compute_credible_interval = traces['delta']
```

Comment:
The confusion is ```cauchy.stan``` only models ```delta```, so we have to set bf threshold on ```delta``` here, which is another inconsistency. But I am not sure what to do. This is just the easiest fix.